### PR TITLE
Update namespace for Allure adapter

### PR DESCRIPTION
**Pull request by okolesnyk**
By adding PHPUnit to namespace we resolving issue: https://github.com/allure-framework/allure-phpunit/issues/41

### CODE REVIEW

**Comment by okorshenko** [comment]
Thank you for fixing the issue. Looking forward for this solution to be merged! 👍

File: src/Yandex/Allure/PHPUnit/Adapter/AllureAdapter.php [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?php
 
-namespace Yandex\Allure\Adapter;
+namespace Yandex\Allure\PHPUnit\Adapter;
```
Comment (baev) [inline src/Yandex/Allure/PHPUnit/Adapter/AllureAdapter.php]: I would like to have `Yandex\Allure\PHPUnit` namespace instead as well as class named `AllurePHPUnit` (or `PHPUnit` replaced with `PhpUnit` in both cases if it is applicable in PHP world)

Comment (okorshenko) [inline src/Yandex/Allure/PHPUnit/Adapter/AllureAdapter.php]:
```suggestion
namespace Yandex\Allure\PHPUnit;
```

Comment (okorshenko) [reply src/Yandex/Allure/PHPUnit/Adapter/AllureAdapter.php]: @baev does it look better now? (can you see suggested code?)

Comment (okorshenko) [reply src/Yandex/Allure/PHPUnit/Adapter/AllureAdapter.php]: I like `PhpUnit ` more than `PHPUnit ` :)

Comment (baev) [reply src/Yandex/Allure/PHPUnit/Adapter/AllureAdapter.php]: 👍 

PS Any thoughts on replacing `PHPUnit` with `PhpUnit`?

Comment (okorshenko) [reply src/Yandex/Allure/PHPUnit/Adapter/AllureAdapter.php]: @okolesnyk will update the code shortly 
@baev thank you for review

**Comment by okorshenko** [comment]
Hi @okolesnyk Thank you for quick fixes
Looks good from my side

Waiting for @baev feedback

**Comment by baev** [comment]
@okolesnyk @okorshenko thanks!

**Comment by baev** [comment]
if there any other updates needed (like dependency updates) feel free to submit PR. Also it would be really appreciated if you guys review https://github.com/allure-framework/allure-phpunit/pull/38 

I'll release the changes tomorrow

**Comment by okorshenko** [comment]
Hi @baev 
Looking forward for new release!
I will review https://github.com/allure-framework/allure-phpunit/pull/38 today. From the glance view - looks reasonable, but I want to double-check rules on versioning

**Pull request merged**